app/views/podcast_planner/_form_side.html.erb

```diff
@@ -2,47 +2,55 @@
   <div class="card-header-primary">
     <h5 class="card-title"><%= t(".header.preview") %></h5>
   </div>
-  <%= turbo_frame_tag "preview", target: "_top" do %>
```

Comment:
moved the turbo frame just around the calendars, which also helps to prevent the `publish_time` and `audio_segments` values being refreshed unnecessarily.

Reply:
+1 good call